src/components/shared/forms/summarySection.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { joinClasses } from 'utilities';
+import { SectionHeader } from 'components/shared';
+
+import './styles/summarySection.css';
+
+export const SummarySection = (props) => (
+  <div className={joinClasses('summary-section', props.className)}>
```

Comment:
I think you need to set a default value for the className or do className || ''. I you look at the rendered element I think you see the `class="summary-section undefined"`

Reply:
Maybe, but in looking at every other component that does this... NONE of them do it including all the newer flyout related ones.